### PR TITLE
Rules Re-Design

### PR DESCRIPTION
**Pull request by BryanRumsey**

### CODE REVIEW

File: client/views/rules-editor.js [hunk 1]
```diff
@@ -31,46 +31,78 @@ module.exports = View.extend({
...
+    $(function () {
       $('[data-toggle="tooltip"]').tooltip();
       $('[data-toggle="tooltip"]').click(function () {
         $('[data-toggle="tooltip"]').tooltip("hide");
       });
     });
   },
   toggleAddRuleButton: function () {
-    this.renderRules();
+    // this.renderEditRules();
```
Comment (jtcooper10) [inline client/views/rules-editor.js:105]: Uncomment this

Comment (jtcooper10) [reply client/views/rules-editor.js:105]: Also replace `$(document).ready()` call on line 114

**Review by jtcooper10 — CHANGES_REQUESTED** [review]
Left two comments. Everything else works. Tested with Chrome and Edge on Windows 10.

File: client/views/rules-editor.js (hunk 1 above)
Comment (BryanRumsey) [reply client/views/rules-editor.js]: Done and done

**Review by jtcooper10 — APPROVED** [review]

**Pull request merged**